.github/workflows/ci.yaml

```diff
@@ -406,3 +406,28 @@ jobs:
         with:
           name: release-images
           path: ./release-images
+
+  trivy:
+    runs-on: ubuntu-18.04
+    needs: prebuild
+    steps:
+      - name: Checkout code
+      - uses: actions/checkout@v2
+
+      - name: Build an image from Dockerfile
+        run: |
+          docker build -t codercom/code-server:${{ github.sha }} ci/release-image/Dockerfile
+
+      - name: Run Trivy vulnerability scanner in repo mode
+        uses: aquasecurity/trivy-action@master
```

Comment:
you may want to be careful with using `@master` because it can break unexpectedly, I'd suggest using the latest current tag version and letting dependabot open PRs for updates

Reply:
Ah, good point! 

Would that be changing it to `aquasecurity/trivy-action@v0.0.14` if that's the [latest release](https://github.com/aquasecurity/trivy-action/releases/tag/0.0.14)?

Reply:
Ah, looking at their code, it looks like it runs trivy inside a Docker container and just runs the latest version: https://github.com/aquasecurity/trivy-action/blob/0.0.14/Dockerfile - so pinning the Action doesn't pin the dependency on `trivy` :frowning: 

We can just accept the risk of stuff breaking suddenly, I just have... some scars from that :joy:

Reply:
Ahh, okay. I will take on the burden so you don't have to relive the scars if this breaks 😂